lib/endpoint.js

```diff
@@ -185,10 +187,11 @@ var Endpoint = function (client) {
 	 * // Callback
 	 * client.Endpoint.createAuthToken("domainId", "endpointId", function (err, endpoint) {});
```

Comment:
Done

2016-07-26 16:18 GMT+03:00 Daniel Tolbert <notifications@github.com>:

> In lib/endpoint.js
> <https://github.com/bandwidthcom/node-bandwidth/pull/88#discussion_r72246302>
> :
>
> > @@ -185,10 +187,11 @@ var Endpoint = function (client) {
> >  	 * // Callback
> >  	 * client.Endpoint.createAuthToken("domainId", "endpointId", function (err, endpoint) {});
>
> update example
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bandwidthcom/node-bandwidth/pull/88/files/2b61d4d86b9d446429a6690e3f8394db64bca36f#r72246302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC6MnLiYyNV3jrfrpIMX4OcJ2YM-t51qks5qZgkfgaJpZM4JVIhM>
> .
>